docs/pt/docs/reference/background.md

```diff
@@ -0,0 +1,11 @@
+# Tarefas em Segundo Plano - `BackgroundTasks`
+
+Você pode declarar um parâmetro em uma *função de operação de caminho* ou em uma função de dependência com o tipo `BackgroundTasks`, e então utilizá-lo para agendar a execução de tarefas em segundo plano após o envio da resposta.
```

Comment:
Most translations are translating _path operation function_ to _**função de operação de rota**_.

Reply:
Thanks! Translation updated in this commit [86f3a46](https://github.com/tiangolo/fastapi/pull/11849/commits/86f3a466a2ca233a85ea221ec509f41a2e860c94)